### PR TITLE
feat: add "copy all" for Paste Options dialog

### LINKED ISSUE #3413: Mark/Unmark all option in the "Pasting layers" dialog
### Description
I would like to suggest a small improvement to "Pasting layers" dialog.
What bothers me is that there's no "Mark all/Unmark all" button. And when you copy layers with exported values you most likely want to copy everything or to leave everything as it is. I always pick the later. So having a button that would unmark everything would be really handy instead of manually clicking on every entry.

### Mock-up
Here's a quick mock-up:

PastingLayers

Button in the red square marks/unmarks all entries in the dialog window.

### PR DESCRIPTION
**Pull request by rodolforg**
fix #3413

### CODE REVIEW

**Comment by rodolforg** [comment]
@Svarov-RZM could you please check it out?

File: synfig-studio/src/gui/dialogs/dialog_pasteoptions.cpp [hunk 1]
```diff
@@ -77,6 +78,21 @@ Dialog_PasteOptions::Dialog_PasteOptions(Gtk::Dialog::BaseObjectType* cobject, c
...
+		Gtk::manage(widget_copy_column_header);
+		auto copy_column_obj = refGlade->get_object("copy_column");
+		if (auto copy_column = Glib::RefPtr<Gtk::TreeViewColumn>::cast_dynamic(copy_column_obj)) {
+			copy_column->set_widget(*widget_copy_column_header);
+			if ((copy_all_checkbutton = dynamic_cast<Gtk::CheckButton*>(widget_copy_column_header))) {
+				copy_all_checkbutton->signal_toggled().connect(sigc::mem_fun(*this, &Dialog_PasteOptions::on_copy_all_toggled));
+				copy_column->signal_clicked().connect([=]() { copy_all_checkbutton->activate(); } );
+			}
+		}
+		widget_copy_column_header->signal_button_release_event().connect([](GdkEventButton*) -> bool {synfig::error("Clicou!"); return false; } );
```
Comment (ice0) [inline synfig-studio/src/gui/dialogs/dialog_pasteoptions.cpp:94]: > synfig::error("Clicou!");

Is this debug output?

Comment (rodolforg) [reply synfig-studio/src/gui/dialogs/dialog_pasteoptions.cpp:94]: Yes XD

**Comment by Svarov-RZM** [comment]
> could you please check it out?

Works exactly like I imagined. Thank you!

File: synfig-studio/src/gui/dialogs/dialog_pasteoptions.cpp (hunk 1 above)
Comment (rodolforg) [reply synfig-studio/src/gui/dialogs/dialog_pasteoptions.cpp]: Fixed! :)

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**